modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -13222,6 +13293,27 @@ private void _testPostCustomObjectEntryWithAttachmentField(
 		Assert.assertEquals("BAD_REQUEST", jsonObject.get("status"));
 	}
 
+	private void _testPostCustomObjectEntryWithNestedPermissions(
+			JSONArray expectedJSONArray, JSONObject objectEntryBodyJSONObject)
+		throws Exception {
```

Comment:
Can you rename it to: `JSONArray expectedPermissionsJSONArray, JSONObject objectEntryJSONObject` ?